pipeline.py

```
@@ -257,6 +257,114 @@ def _mistral_ocr(image) -> OcrResult:
         LOGGER.warning("Mistral OCR failed: %s", exc)
         return OcrResult("", [], [])
 
+def _datalab_ocr(image) -> OcrResult:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)